source/_patterns/04-components/tag/tag.twig

```diff
@@ -3,6 +3,6 @@
   type ? 'tag--' ~ type,
   size ? 'tag--' ~ size
 ] %}
-<a href="{{url}}" class="{{ classes|join(' ')}}" rel="tag">
-  {{title}}
-</a>
\ No newline at end of file
+<a href="{{ url }}" class="{{ classes|join(' ')}}" rel="tag">
```

Comment:
I think you need a space after the closing ) in the join call, before the }} as well.